src/main/resources/anniv5Resources/localization/eng/siegepack/Cardstrings.json

```diff
@@ -0,0 +1,59 @@
+{
+    "${ModID}:Shelling": {
+      "NAME": "Shelling",
+      "DESCRIPTION": "Deal !D! damage, and !${ModID}:sd! damage to a random other enemy. NL Gain 1 ${ModID}:Shell.",
+      "FLAVOR": "General purpose insect repellent."
+    },
+    "${ModID}:Suppress": {
+      "NAME": "Suppress",
+      "DESCRIPTION": "Deal !D! damage. NL Apply !M! *Weak. NL Target loses 3 Strength this turn.",
```

Comment:
don't type the numbers if they're a variable 
also Weak is highlighted here why?